### ISSUE TITLE
Preview url of PDF does not work in Office Add-in / Edge 

### ISSUE THREAD

**Issue by tomoemp** [OneDrive/onedrive-api-docs#1313]
- [ ] Question
- [ ] Documentation issue
- [x] Bug

#### Expected or Desired Behavior
We can see the preview of a pdf when we navigate to the preview link in at least Chrome, Internet Explorer, Edge (both older ver. and chronium Edge) and Firefox, as long as access_token is valid.

#### Observed Behavior

In Edge and in Office add-in for Windows 10 ver. >= 1903 / Office 365 ver >= 16.0.11629, the preview doesn't show up. 
Occasionally, the link works and shows preview once, but only once.

We get "Hmm... looks like this file doesn't have a preview we can show you" message.
When we open the link in Chrome, Internet Explorer and Firefox, the preview shows up without any problem.

(Opened in Edge)
![image](https://user-images.githubusercontent.com/25950494/86935365-076d7080-c13d-11ea-9089-ccd20df7db25.png)

(Opened in Chrome)
![image](https://user-images.githubusercontent.com/25950494/86935152-ce350080-c13c-11ea-85f5-f8bd348fc714.png)

(In Console in Edge)
![image](https://user-images.githubusercontent.com/25950494/86938764-10604100-c141-11ea-99af-9ecd7f36b335.png)

#### Steps to Reproduce
POST request to : https://graph.microsoft.com/v1.0/drives/{driveId}/items/{itemId}/preview
Navigate to "getUrl" in the response in Edge.
If the link works, open an another tab, and open the link again.

(Edge version used)
Microsoft Edge 44.18362.449.0
Microsoft EdgeHTML 18.18362

(example of response)
{"@odata.context":"https://graph.microsoft.com/v1.0/$metadata#microsoft.graph.itemPreviewInfo",
"getUrl":"https://{tenant}/sites/{sitename}/_layouts/15/embed.aspx?uniqueId={uniqueId}&access_token={access_token}",
"postParameters":null,
"postUrl":null}

**Comment by patrick-rodgers** [comment]
As part of a repository clean up effort we are closing older issues. If this issue remains, please: open a new issue, reference this issue, and provide any additional details that may help in resolution. Thank you for your understanding as we work to improve our responsiveness.
